ReactAndroid/src/main/java/com/facebook/react/views/recyclerview/RecyclerViewBackedScrollView.java

```diff
@@ -294,7 +297,8 @@ public int getItemCount() {
 
     @Override
     public long getItemId(int position) {
-      return mViews.get(position).getId();
+      // this should return the view's actual ID to work properly with the view we're extending.
```

Comment:
What does this comment mean? Does it mean this should return the id (`View.getID`) but in fact it doesn't?